lxd/containers_post.go

```diff
@@ -165,106 +160,21 @@ func createFromImage(d *Daemon, req *containerPostReq) Response {
 
 	_, err = dbCreateContainer(args)
 	if err != nil {
-		removeContainerPath(d, name)
 		return SmartError(err)
 	}
 
 	c, err := newLxdContainer(name, d)
 	if err != nil {
-		removeContainer(d, name)
 		return SmartError(err)
 	}
```

Comment:
Why don't we need this cleanup anymore now? (both removeContainerPath and removeContainer) ^